### ISSUE TITLE
[Feature]: Open Source ?

### ISSUE THREAD

**Issue by aroooo** [odioapp/odio#116]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
Make this project open source !
I've seen duplicates but you never responded or made a statement about that.
@spiral2k would you consider making this app open source ?

**Comment by cr7yash** [comment]
Yes, I'll be thankful if the creator can find a solution to help with resources to build such a great app like odio without making it an open source app. I am very intrigued to learn.

**Comment by KhanFromChedum** [comment]
@cr7yash Indeed. In top of that, you can find a pinned tweet on the odio twitter account where a man is thankfull about odio being 'open source'.  
I've tried to unpack the asar file to get more info about the source code and surprise : it's obfuscated !
I began to have a strange feeling about this app, and it's not good...

In my opinion, the author must assume he want his app being closed source and stop messing everyone about it.
